networkx/algorithms/community/modularity_max.py

```diff
@@ -15,22 +15,30 @@
 ]
 
 
-def greedy_modularity_communities(G, weight=None):
-    """Find communities in graph using Clauset-Newman-Moore greedy modularity
-    maximization. This method currently supports the Graph class and does not
+def greedy_modularity_communities(G, weight=None, resolution=1):
+    """Find communities in G using greedy modularity maximization.
+
+    This function uses Clauset-Newman-Moore greedy modularity maximization [2]_.
+    This method currently supports the Graph class and does not
     consider edge weights.
 
     Greedy modularity maximization begins with each node in its own community
     and joins the pair of communities that most increases modularity until no
     such pair exists.
 
+    This function is able to maximize the generalized modularity with
+    resolution parameter $\gamma$.
```

Comment:
I personally found this confusing - this refers to the `resolution` parameter, but references an equation in a separate function docstring. IMO this would be more clear if either:
 1. The equation expressing modularity was repeated in this docstring, or
 2. Reword to make it more explicit that `resolution` is $\gamma$, e.g.
    > This function maximizes the generalized modularity, where `resolution`
        is the resolution parameter, often expressed as $\gamma$ in the
        generalized modularity